arrows/core/derive_metadata.cxx

```diff
@@ -210,47 +237,86 @@ double
 compute_gsd( kwiver::vital::metadata_sptr const& metadata,
              size_t frame_width, size_t frame_height )
 {
-  if ( frame_width == 0 || frame_height == 0 )
+  if( frame_width < 1 || frame_height < 1 )
   {
-    VITAL_THROW( kv::invalid_value, "frame dimensions cannot be zero" );
+    VITAL_THROW( kv::invalid_value, "frame dimensions must both be positive" );
```

Comment:
Pedantically, while not "wrong", this change is unnecessary.

Reply:
It was to match the fact that negative values should be excluded. Maybe I'm missing something.

Reply:
> Maybe I'm missing something.

...like the fact that these values are `size_t`, which is an `unsigned` type and therefore cannot *possibly* be negative? 🙂

OTOH, the change makes the code safe against the type possibly changing, which is why it's not "wrong", just "*pedantically* unnecessary". IOW, this is an observation, not a suggestion.